kalite/coachreports/views.py

```diff
@@ -192,13 +192,15 @@ def tabular_view(request, report_type="exercise"):
     """Tabular view also gets data server-side."""
     # important for setting the defaults for the coach nav bar
 
+    language = request.language
```

Comment:
We should convert the language code in here to the right format through `lc_to_django_lang`, so that all the other functions below can just assume that it's been processed already.